src/apps/integrations/oneup_health/service/task.py

```diff
@@ -43,7 +47,12 @@ def _exponential_backoff(retry_count) -> int:
 
 
 async def _process_data_transfer(
-    session, applet_id: uuid.UUID, unique_id: uuid.UUID, oneup_user_id: int, start_date: datetime | None
+    session,
+    applet_id: uuid.UUID,
+    unique_id: uuid.UUID,
```

Comment:
Same comment here and below…

```suggestion
    submit_id: uuid.UUID,
```